apps/prairielearn/elements/pl-drawing/pl-drawing.js

```diff
@@ -193,6 +193,9 @@ window.PLDrawingApi = {
         if (image_tooltip !== null) {
           btn.setAttribute('title', image_tooltip);
         }
+        if (!elem_options.editable) {
+          btn.disabled = true;
+        }
```

Comment:
We could have done this directly in the markup that's rendered from the Python code, but it was marginally easier to just do this here in JS.